ndindex/booleanarray.py

```diff
@@ -101,7 +101,7 @@ def count_nonzero(self):
         2
         """
         from numpy import count_nonzero
-        return count_nonzero(self.array)
+        return int(count_nonzero(self.array))
```

Comment:
Not sure what this will do in the case where an `array` is returned...
https://numpy.org/doc/2.2/reference/generated/numpy.count_nonzero.html